frontend/src/components/ViewTrips/view-activities-button.js

```diff
@@ -2,11 +2,16 @@ import React from 'react';
 import { useHistory } from 'react-router-dom';
 
 import Button from 'react-bootstrap/Button';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 import { VIEW_ACTIVITIES } from '../../constants/routes.js';
 
 /**
- * Component used for the ViewTrips page to redirect to the ViewActivities page.
+ * Component that redirects the user from the ViewTrips page to the
+ * ViewActivities page corresponding to the trip the button belonged to.
+ *
+ * @property {Object} props These are the props for this component:
```

Comment:
Is this supposed to end in a colon?

Reply:
That is how I have been defining props in JSDoc in all my other files. So in terms of consistency yes.